rio_tiler/models.py

```diff
@@ -0,0 +1,69 @@
+"""rio-tiler models."""
+
+from enum import Enum
+from typing import Dict, List, Optional, Tuple, Union
+
+from pydantic import BaseModel
+
+NumType = Union[float, int]
+BBox = Tuple[NumType, NumType, NumType, NumType]
+ColorTuple = Tuple[int, int, int, int]
+
+
+class NodataTypes(str, Enum):
+    """rio-tiler Nodata types."""
+
+    Alpha = "Alpha"
+    Mask = "Mask"
+    Internal = "Internal"
+    Nodata = "Nodata"
+    Empty = "None"
+
+
+class Bounds(BaseModel):
+    """Dataset Bounding box"""
+
+    bounds: BBox
+
+
+class SpatialInfo(Bounds):
```

Comment:
Do we want to make these dict-like to be backwards compatible?  Similar to the tile responses.

Reply:
yes it will be better... I assume it wasn't possible but didn't check :D

Reply:
@geospatial-jeff can you check at https://github.com/cogeotiff/rio-tiler/pull/297/commits/dda29be27d249d0e644cc23125280e34facebe94 to see if I'm not doing something stupid? 😄